doctest/parts/doctest.cpp

```diff
@@ -290,10 +301,10 @@ namespace detail {
         //unsigned int getElapsedMilliseconds() const {
         //    return static_cast<unsigned int>(getElapsedMicroseconds() / 1000);
         //}
-        double getElapsedSeconds() const { return getElapsedMicroseconds() / 1000000.0; }
+        double getElapsedSeconds() const { return (getCurrentTicks() - m_ticks) / 1000000.0; }
```

Comment:
shouldn't we keep the division by 1 million? I don't see ```getCurrentTicks()``` being changed in terms of behavior...